leetcode/src/1653.c

```diff
@@ -0,0 +1,29 @@
+#define min(X, Y)  ((X) < (Y) ? (X) : (Y))
+
+// Dynamic programming approach. Down -> Up.
+// Runtime: O(n)
+// Space: O(1)
+int minimumDeletions(char * s){
+    int len = strlen(s);
+    
+    int aStateValue = s[0] == 'b';
+
+    int bStateValue = 0;
+    
+    int newAStateValue;
+    int newBStateValue;
+
+    for(int i = 1; i < len; i++){
+        newAStateValue = aStateValue + (s[i] == 'b');
+
+        newBStateValue = min(
+                               aStateValue,
+                               bStateValue + (s[i] == 'a')
+                             );

```

Comment:
Non-blocking - this formatting feels weird. @Panquesito7 Are we using any configurable linters for this repo? I couldn't figure out where the linting analysis exists (in the workflow files). PS: I ask because these should be standardised for the repo - so that we don't have to worry about styling 😄

Reply:
I believe `clang-tidy` and `clang-format` are our linters/formatters.

https://github.com/TheAlgorithms/C/blob/master/.github/workflows/awesome_workflow.yml#L11
https://github.com/TheAlgorithms/C/blob/master/.clang-format
https://github.com/TheAlgorithms/C/blob/master/.clang-tidy

Reply:
Cool, I'll look into this over the weekend to see if we can configure some auto formatting for people 😄 Feel free to merge this PR for now